admin-ui/plugins/auth-server/components/Clients/ClientListPage.js

```diff
@@ -45,11 +45,14 @@ function ClientListPage({ clients, permissions, scopes, loading, dispatch }) {
   const options = {}
   const myActions = []
   const history = useHistory()
+  const { search } = useLocation();
   const pageSize = localStorage.getItem('paggingSize') || 10
   const theme = useContext(ThemeContext)
   const selectedTheme = theme.state.theme
   const themeColors = getThemeColor(selectedTheme)
   const bgThemeColor = { background: themeColors.background }
+  const [scopeClients, setScopeClients] = useState()
+  const [haveINUMParam] = useState(search.indexOf('?inum=') > -1)
```

Comment:
please use `?associatedClients=true` query params. 

`inum=` is not clear.

Reply:
yes, but I still need inum to get clients of scope @duttarnab

Reply:
So, if it navigate from Scope List page, no need to fetch data from API again, since it's already in Scope data (redux)

Reply:
ok. got it. Can we change query param name to `scopeInum`